src/reference/asciidoc/amqp.adoc

```diff
@@ -1058,23 +1058,32 @@ public IntegrationFlow pubSubInFlow(ConnectionFactory connectionFactory) {
 [[amqp-message-headers]]
 === AMQP Message Headers
 
-The Spring Integration AMQP Adapters will map standard AMQP properties automatically.
+The Spring Integration AMQP Adapters will map all AMQP properties and headers automatically.
+(This is a change in 4.3 - previously, only standard headers were mapped).
 These properties will be copied by default to and from Spring Integration `MessageHeaders` using the
 http://docs.spring.io/spring-integration/api/org/springframework/integration/amqp/support/DefaultAmqpHeaderMapper.html[DefaultAmqpHeaderMapper].
 
-Of course, you can pass in your own implementation of AMQP specific header mappers, as the adapters have respective properties to support that.
+Of course, you can pass in your own implementation of AMQP specific header mappers, as the adapters have respective
+properties to support that.
 
-Any user-defined headers within the AMQP http://docs.spring.io/spring-amqp/api/org/springframework/amqp/core/MessageProperties.html[MessageProperties] will NOT
-be copied to or from an AMQP Message, unless explicitly specified by the _requestHeaderNames_ and/or
+Any user-defined headers within the AMQP http://docs.spring.io/spring-amqp/api/org/springframework/amqp/core/MessageProperties.html[MessageProperties] WILL
+be copied to or from an AMQP Message, unless explicitly negated by the _requestHeaderNames_ and/or
 _replyHeaderNames_ properties of the `DefaultAmqpHeaderMapper`.
+For an inbound mapper, the deliveryMode and all `x-*` headers are not mapped by default.
```

Comment:
The `deliveryMode` comment can now be removed; that was from before AMQP pull 387

> For an inbound mapper, `x-*` headers are not mapped by default.